lua/astrocommunity/colorscheme/oldworld-nvim/README.md

```diff
@@ -0,0 +1,10 @@
+# oldworld.nvim
+
+OldWorld is a theme heavily based on mellow.nvim but adding new colors and new integrations with different plugins.
+
+**Repository:** <https://github.com/dgox16/oldworld.nvim>
+
+```vim
+" Vim Script
+colorscheme oldworld
+```
```

Comment:
This can be removed

Reply:
extra docs are fine, not too bad

Reply:
@mehalter  Then shouldn't we instead add extra docs about how to set it up the astro way?